docs/eventing/samples/kafka/channel/README.md

```diff
@@ -151,3 +151,72 @@ Now you can see the events in the log of the `ksvc` using the command:
 ```
 kubectl logs --selector='serving.knative.dev/service=broker-kafka-display' -c user-container
 ```
+
+## Authentication against an Apache Kafka broker
+
+The KafkaSource supports TLS and SASL authentication methods.
```

Comment:
It's not clear to me from this how this actually works; the title mentions the broker but the description mentions the source. Can we get some more detail about what TLS and SASL mean in this context, why would someone use it, how do the broker and source fit in?

Reply:
Apache Kafka - the message broker (I will remove the broker, since I am NOT talking Knative Broker here)

Reply:
I still don't think it's clear what is actually being authenticated against what here, or why

Reply:
OK, not sure how to better say it.

"If Apache Kafka is running in a secured mode, for instance using TLS or SASL, this section show how to confiugure the channel to work against a protected Apache Kafka cluster" ?